src/stores/AuthStore.js

```diff
@@ -84,17 +82,33 @@ class AuthStore {
 
   @computed
   get isAuthorized(): boolean {
-    return !!this._authState.token;
+    return !!this._authState.accessToken;
   }
 
   _rehydrateState = async (): Promise<void> => {
-    const authState = await Storage.get(AUTH_STORAGE_KEY);
-    runInAction(() => {
-      if (authState && authState.token) {
-        this._authState = authState;
+    try {
+      const authState = await Storage.get(AUTH_STORAGE_KEY);
+      let newAuthState = authState;
+
+      if (authState && authState.refreshToken) {
+        const newAuthResponse = await DAOApi.Auth.refreshToken(
+          authState.refreshToken,
+        );
+        newAuthState = { ...authState, ...newAuthResponse };
       }
-      this.isReady = true;
-    });
+
+      runInAction(() => {
+        if (newAuthState && newAuthState.accessToken) {
+          this._authState = newAuthState;
+        }
+      });
+    } catch (error) {
+      // swallow
```

Comment:
This seems like a bad idea

Reply:
when we get an error here, it drops to login page (by reaction triggered by finally case)..I don't think that user should know any reason why error appears during refresh token updating.

Reply:
Yeah, but we should handle it better. If there is an error with the refresh token, it probably means that the auth token is expired too.

Reply:
ok, I'll remove authstate from local storage in catch block